programs/eosc/main.cpp

```diff
@@ -322,11 +322,11 @@ int send_command (const vector<string> &cmd_line)
 
     SignedTransaction trx;
     trx.scope = sort_names({sender,recipient});
-    trx.emplaceMessage(config::EosContractName, vector<types::AccountPermission>{{sender,"active"}}, "transfer",
+    transaction_helpers::emplace_message(trx, config::EosContractName, vector<types::AccountPermission>{{sender,"active"}}, "transfer",
                        types::transfer{sender, recipient, amount});
     auto info = get_info();
     trx.expiration = info.head_block_time + 100; //chain.head_block_time() + 100;
-    trx.set_reference_block(info.head_block_id);
+    transaction_helpers::set_reference_block(trx, info.head_block_id);
```

Comment:
I find the name transaction_helpers long, I feel we should say "using transaction_helpers" unless there is clear possibility of name conflict.